clients/drcachesim/common/options.cpp

```diff
@@ -903,7 +916,9 @@ droption_t<std::string>
     op_cpu_schedule_file(DROPTION_SCOPE_FRONTEND, "cpu_schedule_file", "",
                          "Path with stored as-traced schedule for replay",
                          "Applies to -core_sharded and -core_serial. "
-                         "Path with stored as-traced schedule for replay.");
+                         "Path with stored as-traced schedule for replay.  If specified "
+                         "with a non-zero -skip_to_timestamp, replay is not triggered "
```

Comment:
> replay is not triggered

But after we skip to the given timestamp, we do resume replaying from that point right?

Reply:
No.  Maybe a whole separate option should be used as this kind of overloads this option; but skipping or any change on replay is not supported.  Reworded to "there is no replay" to make this clearer.